browser-test/src/support/index.ts

```diff
@@ -449,9 +449,17 @@ export const validateScreenshot = async (
       // regexp will break and it will show up in screenshots.
```

Comment:
This comment is for line 442-443 (idk why I can't click there)

nit: Move to line 446

Reply:
Done. Also added a comment for the application ID replacement.